### PR TITLE
#89 Opstrace Added the project details in OSSP-master

### PR DESCRIPTION
**Pull request by Aditi8007**

### CODE REVIEW

**Review by asa1997 — CHANGES_REQUESTED** [review]
@Aditi8007 Please add the version details file as well. Also, please resolve the conflicts.

File: bes_theme/assets/data/OSSP-master.js [hunk 1]
```diff
@@ -1691,6 +1691,78 @@ const projectOfInterestObject = {
 			]
 
 		}
+		,
+		{
+			"id": 89,
+			"bes_tracking_id": 89,
+			"issue_url": "https://github.com/Be-Secure/BeSLighthouse/issues/89",
```
Comment (asa1997) [inline bes_theme/assets/data/OSSP-master.js:1698]: Url should point to Be-Secure repo not BeSLighthouse repo.

**Review by asa1997 — CHANGES_REQUESTED** [review]
@Aditi8007 Please address the comment and resolve the conflicts.

**Review by asa1997 — APPROVED** [review]
@Aditi8007 @harimohanr LGTM.
@Aditi8007 Please do update the scores too asap.

**Review by asa1997 — CHANGES_REQUESTED** [review]
@Aditi8007 Please resolve the conflicts

**Review by sudhirverma — COMMENTED** [review]
@Aditi8007 can you move this Pull Request to besecure-osspoi-datastore

**Comment by asa1997** [comment]
Closing this pr as it is outdated.

**Pull request closed without merging**